resources/ts/Components/Integrations/Detail/CredentialsAuthClients.tsx

```diff
@@ -63,26 +70,31 @@ export const CredentialsAuthClients = ({
         {t("details.credentials.uitid_v2")}
       </Heading>
       {keyVisibility === KeyVisibility.v1 ? (
-        <div className="flex flex-col flex-1 gap-4">
-          <div>
-            <Trans
-              i18nKey="details.credentials.uitid_alert"
-              components={[
-                <Link
-                  key={t("details.credentials.uitid_alert")}
-                  href={t("details.credentials.uitid_alert_link")}
-                  className="text-publiq-blue-dark hover:underline mb-3"
-                />,
-              ]}
-            />
+        keyVisibility === KeyVisibility.v1 && !!keyVisibilityUpgrade ? (
```

Comment:
The first part of the condition is already covered by the ternary but more generally I would avoid the nested ternary as I feel it's hard to parse mentally. Could we extract the subcondition to a variable perhaps?